Samples/WinMLSamplesGallery/WinMLSamplesGallery/Pages/SampleBasePage.xaml.cs

```diff
@@ -56,5 +58,12 @@ protected override async void OnNavigatingFrom(NavigatingCancelEventArgs e)
                 page.StopAllEvents();
             }
         }
+
+        public static void SetModelNameForTelemetry(String modelName, String sampleName, LearningModel model)
```

Comment:
>LearningModel model [](http://example.com/codeflow?start=89&length=19)

looks like you need access to the model from c# land. the streaming demo manages the model from the native side, is there a way to get the same telemetry without the specific model instance?